lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -204,6 +203,15 @@ const ItineraryGridSmall = styled.div`
   }
 `
 
+// invisible header rendered for screen readers and a11y technologies
+const InvisibleHeader = styled.span`
+  //place it in an unused grid cell so it doesn't add a row
```

Comment:
Could we adjust this comment to make it clear that this is a bit of a hack? I don't mind it for now but I'd like to have it be a TODO to find a cleaner solution for this